app/utils/clipboard.ts

```diff
@@ -1,18 +1,26 @@
 import Clipboard from "@react-native-community/clipboard"
+import { ApolloClient } from "@apollo/client"
 
 import { validPayment } from "./parsing"
 import { modalClipboardVisibleVar, walletIsActive } from "../graphql/query"
 import { Token } from "./token"
-import { ApolloClient } from "@apollo/client"
+import { loadString } from "./storage"
+import { LAST_CLIPBOARD_PAYMENT } from "../components/modal-clipboard"
```

Comment:
This creates a circular dependency. Better to define the const here so that dependencies can be a DAG.